parsl/tests/manual_tests/htex_local.py

```diff
@@ -0,0 +1,25 @@
+from parsl.providers import LocalProvider
+from parsl.channels import LocalChannel
+# from parsl.launchers import SimpleLauncher
+from parsl.launchers import SingleNodeLauncher
+
+from parsl.config import Config
+from parsl.executors import HighThroughputExecutor
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            label="htex_local",
+            # worker_debug=True,
+            cores_per_worker=1,
+            provider=LocalProvider(
+                channel=LocalChannel(),
+                init_blocks=1,
+                max_blocks=1,
+                tasks_per_node=1,  # For HighThroughputExecutor, this option should in most cases be 1
+                launcher=SingleNodeLauncher(),
+            ),
+        )
+    ],
+    strategy=None,
```

Comment:
Is it still necessary to disable strategy?

Reply:
Not necessary at all.